conf/ups.conf.sample

```diff
@@ -117,3 +135,6 @@
 #
 # To find out if your driver supports any extra settings, start it with
 # the -h option and/or read the driver's documentation.
+
+# Set maxretry to 3 by default, this should mitigate race with slow devices:
+maxretry = 3
```

Comment:
I would recommend putting the global `maxretry` before the sample UPS configuration sections.